teleprox/client.py

```diff
@@ -441,19 +484,21 @@ def ensure_connection(self, timeout=1.0):
                     return
                 except TimeoutError:
                     continue
-            raise TimeoutError(f"Could not establish connection with RPC server at {self.address.decode()}")
+            raise TimeoutError(
```

Comment:
Just curious: are changes like this applied automatically? I am not sure they are improving the code.

Reply:
also, if you had a max line length that you thought better (current: 88), I could alter my black invocation to include that.